app/presenters/order_detail_notice_presenter.rb

```diff
@@ -0,0 +1,85 @@
+class OrderDetailNoticePresenter < DelegateClass(OrderDetail)
+
```

Comment:
I'm not super familiar with how/why DelegateClass (or SimpleDelegator, for that matter) would be used. We're essentially adding additional function onto the order detail object with OrderDetailNoticePresenter as a sort of wrapper class? What is the advantage of doing this delegation rather than having an entirely separate presenter class?